src/components/MAPI/guides/utils.ts

```diff
@@ -0,0 +1,212 @@
+import { IState } from 'stores/state';
+
+export function getCurrentInstallationContextName(state: IState): string {
+  const codeName = state.main.info.general.installation_name;
+
+  return `gs-${codeName}`;
+}
+
+export type KubectlGSCommandModifier = (
+  parts: readonly string[]
+) => readonly string[];
+
+/**
+ * Construct a `kubectl gs` command example.
+ *
+ * You can use modifiers to add arguments or flags to the command.
+ * */
+export function makeKubectlGSCommand(
```

Comment:
I created a new builder function for creating `kubectl gs` examples.

It uses modifiers to add arguments/flags to the commands, and each command has a custom modifier. There's also a modifier for automatically formatting the command output, so it spans multiple lines.